sql/moz-fx-data-shared-prod/search_terms_derived/search_terms_daily_v1/metadata.yaml

```
@@ -0,0 +1,24 @@
+friendly_name: Search Terms Daily
+description: |-
+  Daily count of terminal search terms collected by Merino.
+owners:
+  - ctroy@mozilla.com
+  - rburwei@mozilla.com
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
+      - workgroup:search-terms/aggregated
+labels:
+  incremental: true
+bigquery:
+  time_partitioning:
+    field: merino_date
```

Looking at the query, there is no `merino_date` field
```suggestion
    field: submission_date
```